build.py

```diff
@@ -86,6 +95,10 @@ def main(*argv):
     py_ver_maj = py_ver[0]
     py_ver_min = py_ver[1]
 
+    hash_names = args.hash_names
+    for hn in hash_names:
+        hashlib.new(hn)
```

Comment:
Quick check to make sure the hash actually exists before we have built the whole installer. As `hashlib` just uses `openssl`, this is a pretty good indicator if it will work or not.